test/system/loc-serv_test.go

```diff
@@ -102,6 +102,9 @@ func Test_4g_to_4g_same_zone_userTracking(t *testing.T) {
 		t.Fatal("Subscription failed: ", err)
 	}
 
+        //wait to make sure the subscription got registered
```

Comment:
How is it possible that subscription is not yet registered?

Reply:
NOTE: We should improve the UT timing in general to reduce test duration.

Reply:
Good point... i have to say that it should be but i am not sure why some test case failed randomly. I was hoping the reason why the subscription is not returning a notification is because it was not added yet. But based on UT system test results following these changes... i see not much improvement. I decided to leave it as is because it doesn't hurt... although it might not help either. There should be an activity to lower the time of the system test by putting all the non-notification test case in one bundle.. or at least grouping most of them... This activity should also look into why some test cases fail here and there. Until then.. i suggest leaving this waiting period in the test cases